### ISSUE TITLE
Behavior of remote->click property different with ha true and false

### ISSUE THREAD

**Issue by sjorge** [Koenkk/zigbee2mqtt#2783]
# Bug Report
## What happened
I noticed the behavior of a remote is different if you enable or disable homeassistant property in the configuration.yaml

With ha=true:
```
info  2020-01-19 19:01:06: MQTT publish: topic 'zigbee2mqtt/bedroom/desk_lamp/remote', payload '{"battery":60,"linkquality":47,"click":"off"}'
info  2020-01-19 19:01:06: MQTT publish: topic 'zigbee2mqtt/bedroom/desk_lamp/remote', payload '{"battery":60,"linkquality":47,"click":""}'
info  2020-01-19 19:01:07: MQTT publish: topic 'zigbee2mqtt/bedroom/desk_lamp/remote', payload '{"battery":60,"linkquality":47,"click":"on"}'
info  2020-01-19 19:01:07: MQTT publish: topic 'zigbee2mqtt/bedroom/desk_lamp/remote', payload '{"battery":60,"linkquality":47,"click":""}'
```

With ha=false
```
zigbee2mqtt:info  2020-01-19 19:02:40: MQTT publish: topic 'zigbee2mqtt/bedroom/desk_lamp/remote', payload '{"battery":60,"linkquality":47,"click":"on"}'
zigbee2mqtt:info  2020-01-19 19:02:43" MQTT publish: topic 'zigbee2mqtt/bedroom/desk_lamp/remote', payload '{"battery":60,"linkquality":47,"click":"off"}'
```

## What did you expect to happen
I'm not sure of this is a bug or not, if this behavior is needed for home assistant to work properly, I would have expected this to be documented somewhere. But I could not find it if this is the case already.

Ideally the behavior should not change, but having a blank `click` action is kind of ugly... so I would prefer it not to be made the save if this is indeed needed for ha to work.

## How to reproduce it (minimal and precise)
change the homeassistant configuration property and restart zigbee2mqtt, click a remote's button.

## Debug Info
zigbee2mqtt version: 1.9.0+dev
CC253X firmware version:

**Comment by Koenkk** [comment]
Duplicate of https://github.com/Koenkk/zigbee2mqtt/issues/2144

TLDR; the empty payload is send to trigger a state change in the home assistant sensor so that the entity can be used in automations. When not using HA this is of course not needed.
